install_config/redeploying_certificates.adoc

```diff
@@ -379,6 +379,11 @@ xref:redeploying-etcd-certificates[*_openshift-etcd/redeploy-certificates.yml_*
 by the new etcd CA on etcd peers and master clients. Alternatively, you can use the
 xref:redeploying-all-certificates-current-ca[*_redeploy-certificates.yml_* playbook] to redeploy certificates for {product-title} components in addition to etcd peers and master clients.
 
+[NOTE]
+====
+Red Hat acknowledges that the `etcd` certificate redeployment may result in copying the `serial` to all master hosts.
```

Comment:
s/may/can

Reply:
I would drop "Red Hat acknowledges that" and just state the fact

Reply:
Agreed! Change commited. Thanks!